plugins/janus_videoroom.c

```diff
@@ -2020,6 +2020,8 @@ typedef struct janus_videoroom_subscriber_stream {
 	janus_videoroom_media type;			/* Type of this stream (audio, video or data) */
 	janus_audiocodec acodec;			/* Audio codec this publisher is using (if audio) */
 	janus_videocodec vcodec;			/* Video codec this publisher is using (if video) */
+	char *h264_profile;                 /* H264 profile used for this stream (if video and H264 codec) */
+	char *vp9_profile;                  /* VP9 profile this publisher is using (if video and VP9 codec) */
```

Comment:
Please fix indentation of the comments (you're using spaces, code style requires tabs).

Reply:
👍